extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -911,7 +911,7 @@ describes.realWin(
 
       it('should remove invalid consentStringType', () => {
         const spy = env.sandbox.stub(user(), 'error');
-        const responseMetadata = {'consentStringType': 4};
+        const responseMetadata = {'consentStringType': 400};
```

Comment:
where does 400 come from?

Reply:
This test deliberately uses an invalid consentStringType. Since we added a new type, `4` is no longer invalid and this test will fail. I changed it to 400 to ensure that the type is not a valid one.